sdkproject/Assets/Mapbox/Unity/MeshGeneration/Modifiers/MeshModifiers/ReplaceFeatureCollectionModifier.cs

```diff
@@ -46,18 +46,6 @@ public override void Initialize()
 		{
 			base.Initialize();
 
-			if (_replaceFeatureModifiers != null && _replaceFeatureModifiers.Count > 0)
-			{
-				foreach (var replaceFeatureModifier in _replaceFeatureModifiers)
-				{
-					if (replaceFeatureModifier != null)
-					{
-						replaceFeatureModifier.Clear();
-					}
-				}
-			}
```

Comment:
Why did we remove this code?

Reply:
I am going to replace this code and push. As mentioned in the above comment, this code was removed when we were exploring the idea of mods self-clearing themselves. We have not implemented that solution, so this code should be aded back in. If anyone has objections we can remove/roll back.